djlsp/parser.py

```diff
@@ -443,6 +450,18 @@ def get_tag_hover(self, line, character, match: Match):
                 )
         return None
 
+    def get_context_hover(self, line, character, match: Match):
+        context_name = self._get_full_hover_name(line, character, match.group(2))
+        logger.debug(f"Find context hover for: {context_name}")
+
+        docs = list(doc_str for d in self.create_jedi_script(context_name).help() if (doc_str := d.docstring()))
```

Comment:
I don't like walrus operator in a list comprehension, i rather have extra lines that are easier to read.